XForm/XForm.Web/Index.jsx

```diff
@@ -497,6 +497,8 @@ class Index extends React.Component {
             const defaultFormat = { test: /.?/                  , formatter: cell => cell                                             }
             const formats = [
                 { test: /^https?:\/\//                          , formatter: cell => <a href={cell} target="_blank">{cell}</a>        },
+                { test: /^file:\/\//                            , formatter: cell => <a href={cell.replace("file:///", "vscode://file/")}>{cell}</a>        },
+                { test: /^\w{1,10}:\/\//                        , formatter: cell => <a href={cell} target="_blank">{cell}</a>        },
```

Comment:
Can this one be merged with the 1st rule? If not, make them next to each other.

Reply:
Good idea.